Regressed by https://github.com/KhronosGroup/EGL-Registry/commit/64aa561f1971
Fixed by https://gitlab.com/qemu-project/qemu/-/commit/fbd57c754f32

In file included from ui/egl-helpers.c:21:
include/ui/egl-helpers.h:46:55: error: unknown type name 'Window'
EGLSurface qemu_egl_init_surface_x11(EGLContext ectx, Window win);
                                                      ^
ui/egl-helpers.c:276:55: error: unknown type name 'Window'
EGLSurface qemu_egl_init_surface_x11(EGLContext ectx, Window win)
                                                      ^

--- include/ui/egl-helpers.h.orig	2019-10-01 22:31:45 UTC
+++ include/ui/egl-helpers.h
@@ -43,7 +43,7 @@ void egl_dmabuf_release_texture(QemuDmaBuf *dmabuf);
 
 #endif
 
-EGLSurface qemu_egl_init_surface_x11(EGLContext ectx, Window win);
+EGLSurface qemu_egl_init_surface_x11(EGLContext ectx, EGLNativeWindowType win);
 
 int qemu_egl_init_dpy_x11(EGLNativeDisplayType dpy, DisplayGLMode mode);
 int qemu_egl_init_dpy_mesa(EGLNativeDisplayType dpy, DisplayGLMode mode);
--- ui/egl-helpers.c.orig	2019-10-01 22:31:46 UTC
+++ ui/egl-helpers.c
@@ -273,14 +273,14 @@ void egl_dmabuf_release_texture(QemuDmaBuf *dmabuf)
 
 /* ---------------------------------------------------------------------- */
 
-EGLSurface qemu_egl_init_surface_x11(EGLContext ectx, Window win)
+EGLSurface qemu_egl_init_surface_x11(EGLContext ectx, EGLNativeWindowType win)
 {
     EGLSurface esurface;
     EGLBoolean b;
 
     esurface = eglCreateWindowSurface(qemu_egl_display,
                                       qemu_egl_config,
-                                      (EGLNativeWindowType)win, NULL);
+                                      win, NULL);
     if (esurface == EGL_NO_SURFACE) {
         error_report("egl: eglCreateWindowSurface failed");
         return NULL;
--- ui/gtk-egl.c.orig	2019-10-01 22:31:46 UTC
+++ ui/gtk-egl.c
@@ -54,7 +54,8 @@ void gd_egl_init(VirtualConsole *vc)
     }
 
     vc->gfx.ectx = qemu_egl_init_ctx();
-    vc->gfx.esurface = qemu_egl_init_surface_x11(vc->gfx.ectx, x11_window);
+    vc->gfx.esurface = qemu_egl_init_surface_x11
+        (vc->gfx.ectx, (EGLNativeWindowType)x11_window);
 
     assert(vc->gfx.esurface);
 }
